stestr/commands/__init__.py

```diff
@@ -0,0 +1,22 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from stestr.commands.failing import failing
+from stestr.commands.init import init
+from stestr.commands.last import last
+from stestr.commands.list import list_command
+from stestr.commands.load import load
```

Comment:
So this has a name conflict because we're overriding the module definition with the function. So when stestr.commands.run goes to call stestr.commands.load.load() things blow up on an attribute error because the function doesn't have a load() attribute.